ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/BenchmarkReporter.scala

```diff
@@ -15,27 +17,12 @@ trait BenchmarkReporter {
 }
 
 object BenchmarkReporter {
-  val toFile = new FileOutputBenchmarkReporter("benchmark.dat")
+  val toFile = new FileOutputBenchmarkReporter(Paths.get("benchmark.dat"))
```

Comment:
In the DAML build, this is gonna end up writing to a Bazel temporary directory. I suggest passing this in through a switch (and if it's not provided, write to STDOUT).